codex-ui/dev/pages/components/Popup.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <PageHeader>
+    Popup
+    <template #description>
+      A component that appears on top of other components. Can include any other component.
+    </template>
+  </PageHeader>
+  <Button
+    secondary
+    data-dimensions="large"
+    @click="show()"
+  >
+    Press here to open popup
+  </Button>
+</template>
+
+<script setup lang="ts">
+import PageHeader from '../../components/PageHeader.vue';
+import { Button, useConfirm } from '../../../src/vue';
+
+const { confirm } = useConfirm();
```

Comment:
Let's create another page for Confirm examle. And Popup page will show anything else inside (for example just a stub text like "Popup can include any component") to demonstrate its nature